.travis.yml

```diff
@@ -34,3 +33,5 @@ script:
 
 after_success:
   coveralls
+  
```

Comment:
Please do not add lines at the end of the file. 
It's highly encouraged to not make any unnecessary white space changes.